client/src/App.js

```diff
@@ -47,19 +49,22 @@ class App extends React.Component {
 
 
   handleGetBookedDates = () => {
-    //http://http://localhost:3004/dates${id}
-    //http://booking-dev2.us-west-1.elasticbeanstalk.com/booking/dates${id}
-    axios.get(`http://localhost:3004/booking/dates${id}`)
+    //http://http://localhost:3004${id}
+    //http://booking-dev2.us-west-1.elasticbeanstalk.com/booking${id}
```

Comment:
I'm assuming these comments are for the purpose of moving between dev and production. Maybe consider implementing a conditional that checks the current environment to manage this dynamically. I think Allen had something for that.